lib/types.ts

```diff
@@ -146,7 +146,7 @@ export type Employee = {
   last_name: string;
   address: Address;
   status: EMPLOYEE_STATUSES;
-  start_date?: string;
+  hiring_date?: string;
```

Comment:
Oh yeah - looking at the [docs](https://www.truework.com/docs/api#retrieve-a-report) and the external API code, it seems we only want to change this for employee, not position.